pymongo/encryption.py

```diff
@@ -358,6 +358,17 @@ def __init__(self, kms_providers, key_vault_namespace, key_vault_client,
               - `aws`: Map with "accessKeyId" and "secretAccessKey" as strings.
                 These are the AWS access key ID and AWS secret access key used
                 to generate KMS messages.
+              - `azure`: Map with "tenantId", "clientId", and "clientSecret" as
+                strings. Additionally, "identityPlatformEndpoint" may also be
+                specified as a string (defaults to 'login.microsoftonline.com').
+                These are the Azure Active Directory credentials used to
+                generate Azure Key Vault messages.
+              - `gcp`: Map with "email" as a string and "privateKey" as a
+                base64 encoded string or `bytes`. Python 2 users must specify
+                "privateKey" as a base64 encoded string. Additionally,
```

Comment:
> Python 2 users must specify "privateKey" as a base64 encoded string.

Why? Can we make this work the same in Python 2 and 3?

Reply:
Changed. I will make the necessary code change in the PR to fix failing EVG tests.